packages/client/test/unit/SigningUtil.test.ts

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable max-len */
 import assert from 'assert'
 import { sign, recover, verify } from '../../src/utils/signingUtils'
-import { toEthereumAddress } from '@streamr/utils'
+import { toEthereumAddress, binaryToHex, hexToBinary } from '@streamr/utils'
 
 const privateKey = '23bead9b499af21c4c16e4511b3b6b08c3e22e76e0591f5ab5ba8d4c3a5b1820'
 
 describe(sign, () => {
     it('produces correct signature', async () => {
         const payload = 'data-to-sign'
         const signature = sign(payload, privateKey)
-        assert.strictEqual(signature, '0x787cd72924153c88350e808de68b68c88030cbc34d053a5c696a5893d5e6fec1687c1b6205ec99aeb3375a81bf5cb8857ae39c1b55a41b32ed6399ae8da456a61b')
+        assert.strictEqual(binaryToHex(signature, true), '0x787cd72924153c88350e808de68b68c88030cbc34d053a5c696a5893d5e6fec1687c1b6205ec99aeb3375a81bf5cb8857ae39c1b55a41b32ed6399ae8da456a61b')
```

Comment:
```suggestion
        assert.strictEqual(signature, hexToBinary('787cd72924153c88350e808de68b68c88030cbc34d053a5c696a5893d5e6fec1687c1b6205ec99aeb3375a81bf5cb8857ae39c1b55a41b32ed6399ae8da456a61b'))
```

(if not supported by `assert`, it would make sense to use `jest` instead)